analyzer/src/main/kotlin/curation/FilePackageCurationProvider.kt

```diff
@@ -70,25 +70,18 @@ class DefaultDirPackageCurationProviderFactory : FilePackageCurationProviderFact
  * specified in [FileFormat].
  */
 class FilePackageCurationProvider(
-    curationFiles: List<File>
-) : SimplePackageCurationProvider(readCurationFiles(curationFiles)) {
-    constructor(curationFile: File) : this(listOf(curationFile))
+    vararg paths: File?
```

Comment:
I would prefer `files` here as well.

Reply:
I'd prefer to stick with `paths` to align with the naming in the config, and to make more clear that these can also be directories.